app/assets/javascripts/components/student_contact.es6.jsx

```diff
@@ -0,0 +1,43 @@
+class StudentPreview extends Component {
```

Comment:
Good catch @unzunz!! @anthonyhbh this is why you're getting rendering issues, The real `student_preview` component is overriding this one since they share the same class name.